packages/app/src/app/components/Preview/DevTools/Tabs/index.tsx

```diff
@@ -69,7 +69,7 @@ const DevToolTabs = ({
               devToolIndex={devToolIndex}
               moveTab={moveTab}
               index={i}
-              key={i}
+              key={pane.id}
```

Comment:
This should be `i` (with an eslint ignore), since `pane.id` is not unique. Leads to small bugs.